push_latest.sh

```diff
@@ -0,0 +1,103 @@
+source venv/bin/activate
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+# -----------------------------------------------------------------------------
+# Validate and build the release.
+# -----------------------------------------------------------------------------
+
+# Affirm on the master branch
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if [ "$BRANCH" != "master" ]; then
+    echo -e "${RED}Not on master branch, run:${NC}"
+    echo "git checkout master"
+    exit
+fi
+
+# TODO(TheDuckCow): Perform check when we have an automated way to set prod env.
+# if [[ `git status --porcelain` ]]; then
+#   echo "There are uncommited changes, ending"
+#   exit
+# fi
+
+git pull --quiet
+echo ""
+echo "Current status (should be empty!)"
+git status
+
+echo "Running tests"
+python run_tests.py -a
+if [ $? -eq 0 ]; then
+    echo "Tests passed, moving on"
+else
+    echo "Tests failed, exiting"
+    exit
+fi
+
+# -----------------------------------------------------------------------------
+# Update the data mapping to the latest values
+# -----------------------------------------------------------------------------
+
+# Manual override: we must recheckout some files after test runs finished,
+# due to some side effects of testing with local code references.
+git checkout MCprep_addon/MCprep_resources/mcprep_data_update.json
+git checkout test_files/test_data/jmc2obj_test_1_15_2.mtl
+git checkout test_files/test_data/mineways_test_combined_1_15_2.mtl
+git checkout test_files/test_data/mineways_test_separated_1_15_2.mtl
+
+python mcprep_data_refresh.py -auto
+
+git status
+
+ANY_DIFF=$(git diff MCprep_addon/MCprep_resources/mcprep_data_update.json)
+if [ -z "$ANY_DIFF" ]
+then
+      echo ""
+else
+      echo "Commit new updates to mapping file before release:"
+      echo $ANY_DIFF
```

Comment:
So I recently learned that Zsh (what macOS uses by default) does allow variables to not have quotes, which is nice. However, if we're assuming this can be run in Bash, quotes are required around variables to prevent issues (cause Bash is weird)
```suggestion
      echo "$ANY_DIFF"
```